charts/jenkins/templates/configuration-as-code-configmap.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.configAsCode }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: jenkin{{ .Values.configAsCode.configMap }}
```

Comment:
To match the volume in the deploymentconfig
```suggestion
  name: {{ .Values.configAsCode.configMap }}
```

Reply:
Good call. The "jenkins" before the variable substitution in the ConfigMap was a typo. I have removed it there so that it matches.